src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exception.DataProcessingException;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
```

Comment:
It looks like the DB_URL is not fully defined here. Make sure to provide the complete database URL string.